app/controllers/application_controller.rb

```diff
@@ -21,6 +21,10 @@ def after_invite_path_for(resource_name)
     users_path
   end
 
+  def to_object(entry_type)
+    entry_type.singularize.camelize.constantize
+  end
+
```

Comment:
this is still used by two other controllers, billables_controller and reports_controller